src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -3760,7 +3760,13 @@ private String saveTempFile(InputStream input) {
                 }
             }
             return null;
-        }
+        } finally
```

Comment:
Looks like this finally block partially duplicates what's in the catch... should just close both in finally?